ports/stm32/boards/Passport/modules/flows/login_flow.py

```diff
@@ -36,6 +38,7 @@ async def enter_pin(self):
     async def check_pin(self):
         (result, _error) = await spinner_task('Validating PIN', login_task, args=[self.pin])
         if result:
+            common.settings.set_volatile('first_four_hash', sha256(self.pin[:4]))
```

Comment:
I would change `first_four_hash` to `pin_prefix_hash` and `check_first_four` to `check_pin_prefix`.